pkg/util/test/util.go

```diff
@@ -1658,14 +1658,16 @@ func GetImagesFromVersionURL(url, k8sVersion string) (map[string]string, error)
 
 // ConstructVersionURL constructs Portworx version URL that contains component images
 func ConstructVersionURL(specGenURL, k8sVersion string) (string, error) {
-	versionURL := path.Join(specGenURL, fmt.Sprintf("version?kbver=%s", k8sVersion))
-	u, err := url.Parse(versionURL)
+	u, err := url.Parse(specGenURL)
 	if err != nil {
 		return "", fmt.Errorf("failed to parse URL [%s], Err: %v", specGenURL, err)
 	}
+	q := u.Query()
+	q.Set("kbver", k8sVersion)
+	u.Path = path.Join(u.Path, "version")
+	u.RawQuery = q.Encode()
```

Comment:
Where were we getting the `///` before? I thought it was `https:///`. If that is the case, it won't be fixed even with your change. Ideally that should be fixed in spawn itself.

Reply:
It was my way of parsing the URL, this fixes it.